compiler-rt/lib/sanitizer_common/symbolizer/scripts/build_symbolizer.sh

```diff
@@ -63,7 +71,15 @@ FLAGS+=" -include ${SRC_DIR}/../sanitizer_redefine_builtins.h -DSANITIZER_COMMON
 LINKFLAGS="-fuse-ld=lld -target $TARGET_TRIPLE"
 
 # Build zlib.
-[[ -d ${ZLIB_BUILD} ]] || git clone https://github.com/madler/zlib ${ZLIB_BUILD}
+if [[ -d ${ZLIB_BUILD} ]]; then
```

Comment:
Should this be `if [[ ! -d ${ZLIB_BUILD} ]]` ?

Reply:
Thanks for catching this, @htyu. Fixed in a follow-up.

```
commit d06596516fcb4d32a26eef5930a50fa56457814b (HEAD -> github/main, github/main)
Author: Davide Italiano <davidino@fb.com>
Date:   Wed Nov 1 18:38:27 2023 -0700

    [build_symbolizer] Fix typo in 2c81d70747ac81b37b6c7639fe7afa328e8f5e79

diff --git a/compiler-rt/lib/sanitizer_common/symbolizer/scripts/build_symbolizer.sh b/compiler-rt/lib/sanitizer_common/symbolizer/scripts/build_symbolizer.sh
index 13acced825e0..520e0a631cb4 100755
--- a/compiler-rt/lib/sanitizer_common/symbolizer/scripts/build_symbolizer.sh
+++ b/compiler-rt/lib/sanitizer_common/symbolizer/scripts/build_symbolizer.sh
@@ -71,7 +71,7 @@ FLAGS+=" -include ${SRC_DIR}/../sanitizer_redefine_builtins.h -DSANITIZER_COMMON
 LINKFLAGS="-fuse-ld=lld -target $TARGET_TRIPLE"
 
 # Build zlib.
-if [[ -d ${ZLIB_BUILD} ]]; then
+if [[ ! -d ${ZLIB_BUILD} ]]; then
   if [[ -z "${ZLIB_SRC}" ]]; then
     git clone https://github.com/madler/zlib ${ZLIB_BUILD}
   else
   ```